provider/github/diff_test.go

```diff
@@ -140,6 +140,45 @@ func TestParseHunks(t *testing.T) {
+
+func TestParseHunks_issue_165(t *testing.T) {
+	require := require.New(t)
+
+	hunks, err := parseHunks(largeHunk)
+	require.NoError(err)
+	for _, hunk := range hunks {
+		hunk.Chunks = nil
+	}
+
+	require.Equal([]*hunk{&hunk{
+		OldStartLine: 6,
+		OldLines:     20,
+		NewStartLine: 6,
+		NewLines:     21,
+	}, &hunk{
+		OldStartLine: 50,
+		OldLines:     7,
+		NewStartLine: 51,
+		NewLines:     7,
+	}, &hunk{
+		OldStartLine: 61,
+		OldLines:     31,
+		NewStartLine: 62,
+		NewLines:     25,
+	}, &hunk{
+		OldStartLine: 97,
+		OldLines:     85,
+		NewStartLine: 92,
+		NewLines:     59,
+	}}, hunks)
+
+	read, hunk, err := parseHunk(largeHunk)
```

Comment:
I'm sorry. We don't use `parseHunk` function anymore. I forgot to remove it in my PR. Could you please remove it?